errors.go

```diff
@@ -128,15 +128,15 @@ func (f *fundamental) Format(s fmt.State, verb rune) {
 	switch verb {
 	case 'v':
 		if s.Flag('+') {
-			io.WriteString(s, f.msg)
+			_, _ = io.WriteString(s, f.msg)
```

Comment:
I know that there are very site that recommend to ignore explicity the errors, but first of all is out of scope the feture in self and that can be a discussion of the way of the library do its standard. So, could you revert all this kind of change around the package?

Reply:
I will revert change like this